setup.py

```diff
@@ -393,6 +393,7 @@ def run(self):
 # ----------------------------------------------------------------------
 # Specification of Dependencies
 
+
```

Comment:
Here

Reply:
thanks! did you do this manually or do you have some magical `git diff` option? 🧙

Reply:
Ah, just manual.